cmd/generate-lint/generate-lint.go

```diff
@@ -161,6 +161,8 @@ func generateTasks() (*shrub.Configuration, error) {
 	group := conf.TaskGroup(lintGroup).SetMaxHosts(maxHosts)
 	group.SetupGroup.Command().Type("system").Command("git.get_project").Param("directory", "gopath/src/github.com/evergreen-ci/evergreen")
 	group.SetupGroup.Command().Function("setup-credentials")
+	cmd := group.SetupGroup.Command().Function("run-make")
+	cmd.Vars = map[string]string{"target": "get-go-imports"}
```

Comment:
Why is it that we need this for evergreen to compile?

Reply:
If you look at all the other targets, `get-go-imports` is required. I'm not really sure, but I think John did something weird that required a specific version of `golang.org/x/tools` to run `goimports` on the codegen'd REST routes but couldn't use glide to install it for some reason.